api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/EligibilityOverrideTestDataFactory.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.dhsc.htbhf.claimant.testsupport;
+
+import uk.gov.dhsc.htbhf.claimant.entity.EligibilityOverride;
+import uk.gov.dhsc.htbhf.dwp.model.EligibilityOutcome;
+
+public class EligibilityOverrideTestDataFactory {
+
+    public static EligibilityOverride aConfirmedEligibilityOverrideWithNoChildren() {
```

Comment:
This EligibilityOverride has nothing to do with children so should be renamed to just `aConfirmedEligibilityOverride`.